embedly/gunicorn.conf

```diff
@@ -1,5 +1,6 @@
 import multiprocessing
 
+access_logfile = "-"
```

Comment:
@relud Well eventually (read: soon) we WILL monitoring the logging, it's just that right now we log certain things in the URL that can be used to reconstruct users histories.  Soon that will no longer be recorded in the URL but rather in a POST body and will want to log requests just for traffic monitoring and performance reasons.  I think by the time it reaches ops to deploy this should already be taken care of and you will just be able to monitor normally.  I will make it very clear in the docs for you to deploy.